README.md

```diff
@@ -254,6 +255,74 @@ plaidClient.getAccounts(access_token, (err, res) => {
 });
 ```
 
+## Payment Initiation
```

Comment:
should this be ### so it's nested under examples?

Reply:
Also should we link the the plaid payment initiation docs?

Reply:
Good suggestions! I'll apply both.

Reply:
It doesn't look like we add specific doc sections for individual products, all of the documentation is just under the `Methods` section. Do we actually need this or can we just link to the actual plaid docs - this info is basically just duplicate there

Reply:
Hey, not totally tracking the suggestion here. But if you don't want to link to the docs from here, I don't think that's a huge deal

Reply:
@jasonjia93 
What is the downside to have additional examples? The examples that are currently in this README are also listed under `Methods`. 

I can remove this section and just add the payment initiation methods under `Methods`. I thought it might help to add the full examples here (where we are not so limited by the size of the code snippets, like in the API docs) to give extra clarity on the use and types of the parameters for `createPaymentRecipient` (iban, bacs and address).

Reply:
One specific example that happened in the past and we'd like to prevent from happening again is a call like `createPaymentRecipient(name, "", address, bacs)`

Reply:
no downside, was just wondering why we weren't doing this for other products!